app/assets/src/components/views/Landing.jsx

```diff
@@ -203,6 +206,22 @@ class Landing extends React.Component {
       </div>
     );
 
+    // const bulletinLink = () => (location.href = "/");
```

Comment:
Why not add this for admins only instead of commenting?
At least we can also see how it is going to look.

What is the purpose of merging this as a comment?

Reply:
But you can put it as a javascript flag, right? Maybe a URL parameter. Not necessarily admin.